### PR TITLE
bugfix (49047): fix error on proposal details page + token decimals issue

### PR DESCRIPTION
**Pull request by maryjanyes**
**Description**:
* Fix error on proposal details page
* Token decimals issue

**Notes for reviewer**:
Needs to be tested carefully before release.

### CODE REVIEW

**Review by web3-nomad — DISMISSED** [review]

File: src/shared/services/MirrorNodeService/MirrorNodeService.ts [hunk 1]
```diff
@@ -127,7 +127,19 @@ function createMirrorNodeService(
...
-    return await nodeAPIs[params.network].get(`/api/v1/tokens/${tokenId}`);
+    try {
+      const response: MirrorNodeTokenById = await nodeAPIs[params.network].get(`/api/v1/tokens/${tokenId}`);
+      const precision = Number(response.data?.decimals) ** 10;
+
+      return { data: { ...response.data, precision } };
+    } catch (err) {
+      return {
+        data: {
+          precision: 1e8,
```
[the remark this answers is not in the document]
Comment (web3-nomad) [reply src/shared/services/MirrorNodeService/MirrorNodeService.ts:138]: at a minimum put it as a non-public constant in the file.

[the remark this answers is not in the document]
Comment (maryjanyes) [reply src/shared/services/MirrorNodeService/MirrorNodeService.ts:138]: > why is this hardcoded?

If no response was coming from token info request:
https://github.com/hashgraph/hedera-accelerator-defi-dex-ui/pull/374/files#diff-255f30ec3fbf213036603de05ccaa0b8aba3078b4973f09a790bbe62b64d16e8R131.

[the remark this answers is not in the document]
Comment (maryjanyes) [reply src/shared/services/MirrorNodeService/MirrorNodeService.ts:138]: > at a minimum put it as a non-public constant in the file.

Which file? Sorry, i am a little bit lost here.

[the remark this answers is not in the document]
Comment (web3-nomad) [reply src/shared/services/MirrorNodeService/MirrorNodeService.ts:138]: in src/shared/services/MirrorNodeService/MirrorNodeService.ts
create a constant that isn't 'extern' whoose value is 1e8. so the next guy coming in knows what it is.
DEFAULT_DEX_PRECISION or something like that.

we removed the global constant to ensure there was nowhere we missed

**Review by web3-nomad — APPROVED** [review]

**Pull request merged**